### DISCUSSION TITLE
can datatable support to change column width by dragging

### DISCUSSION THREAD

**Discussion by rexwangyi** [tusen-ai/naive-ui#3571] [Q&A] [answered]
I want to change column width by dragging header as the element UI does.
Can you help me.

**Comment by Sepush** [comment, marked as answer]
We have a WIP PR for this feature https://github.com/tusen-ai/naive-ui/pull/3462

**Comment by jahnli** [comment]
好像支持了

**Reply by Sepush** [reply]
还有一些需要处理的地方
